sequencer.c

```diff
@@ -2032,6 +2039,20 @@ static int pick_commits(struct todo_list *todo_list, struct replay_opts *opts)
 			*end_of_arg = '\0';
 			res = do_exec(item->arg);
 			*end_of_arg = saved;
+
+                        /* Reread the todo file if it has changed. */
+                        if (res == 0) {
+                                struct stat st;
+                                if ((res = stat(get_todo_path(opts), &st)))
+                                        return res;
```

Comment:
This does not report what went wrong, possibly leaving the user puzzled. Maybe

```c
if (stat(get_todo_path(opts), &st))
    return error_errno(_("could not re-read '%s'"), get_todo_path(opts));
```

would make sense instead?